src/LaunchModalButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+class LaunchModalButton extends React.Component {
```

Comment:
This can be a functional component, because it's stateless

Reply:
It's stateless now, but will have state shortly.  Is it better practice to write functional components and only convert once state is introduced?